Samples/outlook-encrypt-attachments/README.md

```diff
@@ -28,7 +28,7 @@ description: "Use Outlook event-based activation to encrypt attachments, process
 This sample showcases how to use event-based activation in an Outlook add-in when the user composes an email or appointment/meeting request.  It demonstrates how to run tasks based on events that fire when certain data changes when the user:
 
 - adds an attachment to an email or appointment/meeting request
-- adds recipients or distributions lists as required or optional attendeees in a meeting request
+- adds recipients or distributions lists as required or optional attendees in a meeting request
```

Comment:
Per [writing guide](https://learn.microsoft.com/en-us/style-guide/scannable-content/lists), each list item should begin with an uppercase letter and end with a period. This and other items in this list.

```suggestion
- Adds recipients or distributions lists as required or optional attendees in a meeting request.
```